lib/dicon/secure_shell.ex

```diff
@@ -139,20 +139,21 @@ defmodule Dicon.SecureShell do
              # TODO: we need to remove this assertion here as well, once we have a
              # better "streaming" API.
              :ok = :ssh_sftp.write(channel, handle, chunk, @timeout)
-             write_progress_bar(chunk_index / chunk_count)
+             write_spinner(chunk_index, chunk_count)
            end),
-           IO.puts("\n"),
+           IO.write(IO.ANSI.format([:clear_line, ?\r])),
```

Comment:
This will remove the progress bar after spinner is done. Not sure if we want this but I thought it was kind of neat.